### PR TITLE
Improve lighting doc

### PR DESCRIPTION
**Pull request by ombre5733**

### CODE REVIEW

**Comment by akien-mga** [comment]
Commit 94c5166 was already merged in #356, so it should not be in this branch too. Could you rebase the branch on top of the current master to get rid of it? (`git pull --rebase upstream master`).

**Comment by ombre5733** [comment]
Ok, rebased it.

File: tutorials/3d/lighting.rst [hunk 1]
```diff
@@ -50,44 +50,55 @@ Each one has a specific function:
...
+should be used to represent lamps or bulbs or any other light source that originates
+approximately in a point.
 
 .. image:: /img/light_omni.png
 
-The attenuation curve for these kind of lights in nature is computed
-with an inverse-quadratic function that never reaches zero and has
-almost infinitely large values near the emissor.
+In reality, the attenuation of omni light is proportional to the squared distance
+from the point source. This can be easily understood if you imagine a sqhere around
```
Comment (akien-mga) [inline tutorials/3d/lighting.rst]: s/sqhere/sphere/

File: tutorials/3d/lighting.rst [hunk 2]
```diff
@@ -50,44 +50,55 @@ Each one has a specific function:
...
+the omni light with a certain radius. No matter how large the sphere is, the number
+of rays passing through it is always the same. If the radius of the sphere is doubled,
+the area of the sphere increases by four. In other words, the density of rays
+(the number of rays per squere area) decreases quadratically with the distance.
```
Comment (akien-mga) [inline tutorials/3d/lighting.rst]: s/squere/square/

File: tutorials/3d/lighting.rst [hunk 3]
```diff
@@ -50,44 +50,55 @@ Each one has a specific function:
...
 
-This makes them considerably inconvenient to tweak for artists, so
-Godot simulates them with an artist-controlled exponential curve
+Inverse-quadratic attenuation curves are inconvenient for artists: they
+never reach zero and have almost intinitely large values near the emitter.
```
Comment (akien-mga) [inline tutorials/3d/lighting.rst]: s/intinitely/infinitely/

File: tutorials/3d/lighting.rst [hunk 4]
```diff
@@ -106,6 +117,6 @@ Baked light
 -----------
 
 Baked light stands for pre-computed ambient light. It can serve multiple
-purposes, such as baking light emissors that are not going to be used in
+purposes, such as baking light emitters that are not going to be used in
 real-time, and baking light bounces from real-time lights to add more
 realism to a scene (see Baked Light]] tutorial for more information).
```
Comment (akien-mga) [inline tutorials/3d/lighting.rst]: Those `]]` are a leftover from a previous wiki-style link. Maybe you could also fix them to point to the Baked Lights tutorial? (<code>:ref:\`doc_light_baking\`</code>)

**Comment by ombre5733** [comment]
@akien-mga Thanks for proofreading. Fixed with latest commit.

**Pull request merged**